### DISCUSSION TITLE
Admin password

### DISCUSSION THREAD

**Discussion by David-Maran** [timschofield/webERP#202] [Installation Issues]
On install, I read this message : 
`The default user name is 'admin' and it cannot be changed. `

Is it real? What about security issues?

**Comment by timschofield** [comment]
It can be changed after the installation. In fact I normally remove the admin user altogether after installation. As I said in a reply to one of the other issues you brought up, I am in the process of rewriting the installer (I didn't write the current one), and this is one of the changes I am doing..

Thanks for bringing up these issues today

Tim

**Comment by timschofield** [comment]
The new installer allows you to change the admin user ID, and the password.

Thanks
Tim
